netci.groovy

```diff
@@ -235,6 +235,30 @@ commitPullList.each { isPr ->
   }
 }
 
+// True when this is a PR job, false for commit. We generally don't want to run the Loc status
+// check on PRs (as we don't want to block PRs due to unlocalized resources) and never want to
+// bother running it on feature branches.
```

Comment:
So if we do want to enable this for PRs, the intent is you switch this to [false, true]?

Reply:
@jasonmalinowski Yes. I'm not sure if that's the best way to do it, but it's what I could come up with.

Reply:
Why don't we want to run this on PR? Generally the only types of jobs we run on commit only are those for which it's rather redundant to do so. Example being Mac. Mac is a limited resource and Ubuntu jobs cover 99% of what Mac does hence we only do Mac on commit.

Reply:
@jaredpar It is expected that this will fail a lot, as untranslated resources are expected and we don't generally want to block PRs on them.

However, I do want to be able to tell if there are untranslated resources in a release branch (so we know to ask the loc team to take care of it ASAP) and, possibly, block PRs that would require translation.

Reply:
So we are adding a jenkins job that will fail, by design, in the commit queues? That seems wrong.

Reply:
Instead of the result of the build being "success" or "failure", think of it as "all translated" or "not all translated". And remember that I'm the only one who needs to be concerned with the result. :-)

I'm open to other suggestions for how to do this, but I don't want to end up duplicating all of the things Jenkins already knows how to do (sync sources, run jobs, report results, etc.).